datafusion/expr/src/expr_fn.rs

```diff
@@ -885,28 +885,29 @@ nary_scalar_expr!(
 scalar_expr!(DatePart, date_part, part date, "extracts a subfield from the date");
 scalar_expr!(DateTrunc, date_trunc, part date, "truncates the date to a specified level of precision");
 scalar_expr!(DateBin, date_bin, stride source origin, "coerces an arbitrary timestamp to the start of the nearest specified interval");
-scalar_expr!(
+nary_scalar_expr!(
+    ToTimestamp,
+    to_timestamp,
+    "converts a string to a `Timestamp(Nanoseconds, None)`"
```

Comment:
Is this accurate? If so, maybe we can update the other doc comments as well

```suggestion
    "converts a string and optional formats to a `Timestamp(Nanoseconds, None)`"
```